### ISSUE TITLE
Plymouth hangs on boot and freezes whole system (Bugzilla Bug 578)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#578]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 578
Date: 2014-03-15 07:40:06 +0000
From: luca &lt;<agoiza@gmail.com>&gt;
To:   @berolinux
CC:   benatto@mandriva.com.br, @berolinux, @christann404, @itchka, @robxu9, @rugyada, @tpgxyz

Last updated: 2014-04-22 15:27:46 +0000

## Comment 4001
Date: 2014-03-15 07:40:06 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

after latest updates, system won't boot

i've noticed an error during dracut update:

*** Including module: lvm ***
grep: /var/tmp/initramfs.9xFN1b/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory
sed: can't read /var/tmp/initramfs.9xFN1b/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory
Skipping udev rule: 64-device-mapper.rules
Skipping udev rule: 56-lvm.rules

boot process stops after "Stsarted plymouth boot screen"

i'm not able also in recovery mode :(

## Comment 4003
Date: 2014-03-15 07:47:16 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i am able to boot with old kernel ...

## Comment 4004
Date: 2014-03-15 09:36:25 +0000
From: @tpgxyz

Do you use lvm ? If not then this is not harmless.

Please provide a real output while your system is booting in debug mode.

## Comment 4009
Date: 2014-03-15 11:52:35 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i had same problem with another system 64bit after dracut install.

when i have time try and start in debug mode.

## Comment 4010
Date: 2014-03-15 12:38:32 +0000
From: @christann404

I am seeing the same behaviour

Chris

## Comment 4033
Date: 2014-03-16 18:51:17 +0000
From: @berolinux

Seeing it on one of my boxes too...

*** Including module: lvm ***
grep: /var/tmp/initramfs.UCPDYR/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory
sed: can't read /var/tmp/initramfs.UCPDYR/lib/udev/rules.d/69-dm-lvm-metad.rules: No such file or directory

may be normal, I don't use lvm, and it's not installed here -- but possibly the error causes an early abort of other scripts or something.

Also seeing this later on in the dracut process:
*** Pre-linking files ***
Executing /sbin/init U
Couldn't find an alternative telinit implementation to spawn.

Maybe this is the true culprit and 69-dm-lvm-metad.rules is a red herring.

Will run some tests later, can't reboot the box right now (building important stuff).

## Comment 4034
Date: 2014-03-16 18:58:26 +0000
From: @tpgxyz

Sorry i do not see this issue on my 2013.0 with backported dracut.

Please provide debug output by adding these commands to end of grub2 boot line:
rd.debug systemd.log_level=debug systemd.log_target=kmsg log_buf_len=1M

## Comment 4040
Date: 2014-03-16 21:32:11 +0000
From: @berolinux

On my box, it isn't related to the lvm spewage.

My kernel command line ended with around 10 instances of "resume=/dev/sda1 init=/lib/systemd/systemd", removing the extraneous ones fixed it without changing initrd.

Turns out there's even multiple "resume=/dev/sda1 init=/lib/systemd/systemd"s in /etc/default/grub.

Looks like the installkernel script or something invoked by it is doing something strange.

## Comment 4041
Date: 2014-03-16 21:42:59 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i have the same problem with "new" beta iso.

but kernel command line seems ok.

## Comment 4042
Date: 2014-03-16 21:43:50 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 271
kernel command line

> Attached file: foto95.png (image/png, 11694 bytes)
> Description:   kernel command line

## Comment 4043
Date: 2014-03-16 21:49:33 +0000
From: @berolinux

Looks like there are multiple issues then -- my 3.13.6-nrjQL-desktop-1omv boots fine after fixing the multiple resume=... init=... entries.

Can you get a trace of where it stops for you (e.g. by disabling plymouth)?

## Comment 4044
Date: 2014-03-16 21:56:06 +0000
From: @tpgxyz

Booting with plymouth.enable=0 systemd.unit=rescue.target
and then after loging as a root on rescue mode and running
systemctl isolate graphical.target gives working KDE.

Noting interesting when running journalctl --boot=-1

which shows log from last failed boot.

## Comment 4045
Date: 2014-03-16 21:58:57 +0000
From: @tpgxyz

Lastest news, adding plymouth.enable=0 to bootparameters fixes boot.

## Comment 4046
Date: 2014-03-16 22:02:36 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

with 

plymouth.enable=0 systemd.unit=rescue.target systemd.log_level=debug 
systemd.log_target=kmsg log_buf_len=1M

after the prompt i can continue my first install

seems plymouth related?

## Comment 4047
Date: 2014-03-16 22:05:31 +0000
From: @tpgxyz

(In reply to comment #13)

> seems plymouth related?

Seems so, see my comment https://issues.openmandriva.org/show_bug.cgi?id=578#c12

Upstream have added a lot of bugfixes to plymouth, and i had an eye on this to update also our.

## Comment 4048
Date: 2014-03-16 23:15:28 +0000
From: @robxu9

assigning to TPG and elevating

## Comment 4051
Date: 2014-03-17 06:18:33 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

next time keep better account of what is reported and also the work of the beta testers

the issue was reported a few minutes after the release of dracut
widely in time to avoid these bad figures

## Comment 4057
Date: 2014-03-17 08:22:35 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

after latest plymouth updates
i have reinstalled dracut and i noticed this error

*** Including module: plymouth ***
plymouth: symbol lookup error: plymouth: undefined symbol: ply_logger_is_tracing_enabled
plymouth: symbol lookup error: plymouth: undefined symbol: ply_logger_is_tracing_enabled
No default plymouth plugin is set

## Comment 4070
Date: 2014-03-17 11:10:36 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 276
boot_journal

> Attached file: log.boot.txt (text/plain, 119048 bytes)
> Description:   boot_journal

## Comment 4071
Date: 2014-03-17 11:12:34 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #17)
> after latest plymouth updates
> i have reinstalled dracut and i noticed this error
> 
> 
> *** Including module: plymouth ***
> plymouth: symbol lookup error: plymouth: undefined symbol:
> ply_logger_is_tracing_enabled
> plymouth: symbol lookup error: plymouth: undefined symbol:
> ply_logger_is_tracing_enabled
> No default plymouth plugin is set

alpha 2 after all update and reinstall dracut

no plymouth on screen, directly logging screen 

in attachement boot journal

note line 855 
mars 17 11:49:41 localhost.localdomain systemd[1]: Started Setup Virtual Console.
mars 17 11:49:41 localhost.localdomain systemd[1]: Starting Show Plymouth Boot Screen...
mars 17 11:49:41 localhost.localdomain plymouthd[546]: /sbin/plymouthd: symbol lookup error: /sbin/plymouthd: undefined symbol: ply_logger_is_tracing_enabled
mars 17 11:49:41 localhost.localdomain plymouthd[546]: unexpectedly exited with status 127 immediately after starting to daemonize
mars 17 11:49:41 localhost.localdomain systemd[1]: plymouth-start.service: control process exited, code=exited status=1
mars 17 11:49:41 localhost.localdomain systemd[1]: Failed to start Show Plymouth Boot Screen.
mars 17 11:49:41 localhost.localdomain systemd[1]: Unit plymouth-start.service entered failed state.
mars 17 11:49:41 localhost.localdomain systemd[1]: Started Forward Password Requests to Plymouth Directory Watch.
mars 17 11:49:41 localhost.localdomain systemd[1]: Starting Dispatch Password Requests to Console Directory Watch.
mars 17 11:49:41 localhost.localdomain systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
mars 17 11:49:41 localhost.localdomain systemd[1]: Starting Paths.

## Comment 4073
Date: 2014-03-17 11:31:23 +0000
From: @tpgxyz

I'm building new plymouth with disabled code tracing feature.

BTW.

after installing new plymouth always run dracut -f

## Comment 4100
Date: 2014-03-17 14:45:55 +0000
From: @tpgxyz

*** Bug #587 has been marked as a duplicate of this bug. ***

## Comment 4105
Date: 2014-03-17 18:21:13 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

i have installed new plymouth updates and run dracut -f:

plymouth                       0.8.9        0.20140316.2  omv   2014.0 x86_64 
plymouth-plugin-label          0.8.9        0.20140316.2  omv   2014.0 x86_64 
plymouth-plugin-script         0.8.9        0.20140316.2  omv   2014.0 x86_64 
plymouth-scripts               0.8.9        0.20140316.2  omv   2014.0 x86_64 
plymouth-system-theme          0.8.9        0.20140316.2  omv   2014.0 x86_64 
plymouth-utils                 0.8.9        0.20140316.2  omv   

without plymoluth.enable=0 i cannot boot
the system hangs at:
[ ok ]Started configure read-only root support.

i can boot in rescue mode

## Comment 4107
Date: 2014-03-17 19:03:34 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

alpha2 after official update (plymouth, samba etc)
plymouth don't work 

mars 17 19:53:59 localhost.localdomain systemd[1]: Starting Show Plymouth Boot Screen...
mars 17 19:54:00 localhost.localdomain systemd[1]: plymouth-start.service: main process exited, code=exited, status=127/n/a
mars 17 19:54:00 localhost.localdomain systemd[1]: Failed to start Show Plymouth Boot Screen.
mars 17 19:54:00 localhost.localdomain systemd[1]: Unit plymouth-start.service entered failed state.
mars 17 19:54:00 localhost.localdomain systemd[1]: Starting Forward Password Requests to Plymouth Directory Watch.
mars 17 19:54:00 localhost.localdomain systemd[1]: Started Forward Password Requests to Plymouth Directory Watch.
mars 17 19:54:00 localhost.localdomain systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
mars 17 19:54:00 localhost.localdomain systemd[1]: Starting Paths.

## Comment 4108
Date: 2014-03-17 19:05:00 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 281
boot plymouth after update

journalctl -b

> Attached file: log.boot_updating.txt (text/plain, 118219 bytes)
> Description:   boot plymouth after update

## Comment 4109
Date: 2014-03-17 19:09:42 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

in addition

no need  plymoluth.enable=0 

the boot seems normal but no big circle on screen

uname -a
Linux localhost.localdomain 3.13.6-nrjQL-desktop-1omv #1 SMP PREEMPT Thu Mar 13 10:55:37 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

## Comment 4114
Date: 2014-03-18 05:48:57 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

alpha 2
new plymouth update

# rpm -qa --last | grep '18 mars'
postfix-2.10.2-3-omv2014.0.x86_64             mar. 18 mars 2014 06:07:14 CET
plymouth-plugin-script-0.8.9-0.20140316.3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:54 CET
plymouth-utils-0.8.9-0.20140316.3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:53 CET
plymouth-system-theme-0.8.9-0.20140316.3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:53 CET
plymouth-plugin-label-0.8.9-0.20140316.3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:52 CET
plymouth-scripts-0.8.9-0.20140316.3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:51 CET
plymouth-0.8.9-0.20140316.3-omv2014.0.x86_64  mar. 18 mars 2014 06:06:49 CET
postfix-config-standalone-2.10.2-3-omv2014.0.x86_64 mar. 18 mars 2014 06:06:47 CET
lib64postfix1-2.10.2-3-omv2014.0.x86_64       mar. 18 mars 2014 06:06:46 CET

after dracut -f

no plymouth screen, different of comment 23, no scrolling different activations,directly on log screen.
note that at each boot there are scan disk efsck

during dracut:
plymouth: symbol lookup error: plymouth: undefined symbol: ply_logger_is_tracing_enabled
plymouth: symbol lookup error: plymouth: undefined symbol: ply_logger_is_tracing_enabled
No default plymouth plugin is set

in yesterday update I think these comments had disappeared

in boot journal:

 Starting Show Plymouth Boot Screen...
mars 18 06:16:07 localhost.localdomain systemd-fsck[391]: /dev/sda4 : propre, 186883/4464640 fichiers, 2686728/17826560 blocs
mars 18 06:16:07 localhost.localdomain systemd[1]: Started File System Check on Root Device.
mars 18 06:16:07 localhost.localdomain systemd[1]: Starting Remount Root and Kernel File Systems...
mars 18 06:16:07 localhost.localdomain plymouthd[450]: /sbin/plymouthd: symbol lookup error: /sbin/plymouthd: undefined symbol: ply_logger_is_tracing_enabled
mars 18 06:16:07 localhost.localdomain plymouthd[450]: unexpectedly exited with status 127 immediately after starting to daemonize
mars 18 06:16:07 localhost.localdomain kernel: EXT4-fs (sda4): re-mounted. Opts: (null)
mars 18 06:16:07 localhost.localdomain systemd[1]: plymouth-start.service: control process exited, code=exited status=1
mars 18 06:16:07 localhost.localdomain systemd[1]: Failed to start Show Plymouth Boot Screen.
mars 18 06:16:07 localhost.localdomain systemd[1]: Unit plymouth-start.service entered failed state.

see attachement boot and dracut

## Comment 4115
Date: 2014-03-18 05:50:01 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 282
errors in dracut -f

> Attached file: dracut_f.txt (text/plain, 2511 bytes)
> Description:   errors in dracut -f

## Comment 4116
Date: 2014-03-18 05:51:53 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 283
boot after plymouth-0.8.9-0.20140316.3-omv2014.0.x86_64

> Attached file: log.boot_18_3_14.txt (text/plain, 118783 bytes)
> Description:   boot after plymouth-0.8.9-0.20140316.3-omv2014.0.x86_64

## Comment 4117
Date: 2014-03-18 08:03:28 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

# rpm -aq | grep plymouth
plymouth-0.8.9-0.20140316.3-omv2014.0.x86_64

i have no "undefined symbol" during  dracut -f
but plymouthd dumped core :(

# systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: failed (Result: core-dump) since lun 2014-03-17 23:01:40 CET; 9h ago
  Process: 438 ExecStartPost=/bin/plymouth show-splash (code=exited, status=0/SUCCESS)
  Process: 410 ExecStart=/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session (code=exited, status=0/SUCCESS)
 Main PID: 431 (code=dumped, signal=SEGV)

mar 17 23:01:37 localhost.localdomain systemd[1]: Started Show Plymouth Boot Screen.
mar 17 23:01:40 localhost.localdomain systemd[1]: plymouth-start.service: main process exited, code=dumped, status=11/SEGV
mar 17 23:01:40 localhost.localdomain systemd[1]: Unit plymouth-start.service entered failed state.
mar 17 23:01:40 localhost.localdomain systemd-coredump[897]: Process 431 (plymouthd) dumped core.

## Comment 4118
Date: 2014-03-18 08:10:45 +0000
From: @tpgxyz

Please use systemd-coredumpctl on that plymouth dump core and attach output in a text file here.

## Comment 4119
Date: 2014-03-18 10:23:25 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

hope can help:

[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
Core was generated by `@sbin/plymouthd --mode=shutdown --attach-to-session'.
Program terminated with signal 11, Segmentation fault.
#0  0x00007f59330bdfc3 in ply_terminal_is_open () from /lib64/libply-splash-core.so.2
#0  0x00007f59330bdfc3 in ply_terminal_is_open () from /lib64/libply-splash-core.so.2
#1  0x00007f59330c0a54 in ply_seat_open () from /lib64/libply-splash-core.so.2
#2  0x00007f59330b9bb3 in ?? () from /lib64/libply-splash-core.so.2
#3  0x00007f59330ba224 in ?? () from /lib64/libply-splash-core.so.2
#4  0x00007f59330ba7a4 in ?? () from /lib64/libply-splash-core.so.2
#5  0x00007f59330ba90f in ?? () from /lib64/libply-splash-core.so.2
#6  0x00007f59330d1b34 in ply_event_loop_process_pending_events () from /lib64/libply.so.2
#7  0x00007f59330d1b8f in ply_event_loop_run () from /lib64/libply.so.2
#8  0x0000000000405e37 in main ()

## Comment 4120
Date: 2014-03-18 11:06:51 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

with debuginfo installed:

#0  ply_terminal_is_open (terminal=0x0) at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-terminal.c:669
No locals.
#1  0x00007f32215e6a54 in add_text_displays (seat=0x115d500) at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-seat.c:105
        display = <optimized out>
#2  ply_seat_open (seat=seat@entry=0x115d500, renderer_type=renderer_type@entry=PLY_RENDERER_TYPE_FRAME_BUFFER, device=device@entry=0x115d570 "/dev/fb0")
    at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-seat.c:161
        __func__ = "ply_seat_open"
#3  0x00007f32215dfbb3 in create_seat_for_terminal_and_renderer_type (manager=manager@entry=0x1157030, device_path=device_path@entry=0x115d570 "/dev/fb0", 
    terminal=terminal@entry=0x0, renderer_type=renderer_type@entry=PLY_RENDERER_TYPE_FRAME_BUFFER)
    at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-device-manager.c:644
        seat = 0x115d500
        is_local_terminal = <optimized out>
        __func__ = "create_seat_for_terminal_and_renderer_type"
#4  0x00007f32215e0224 in create_seat_for_udev_device (manager=manager@entry=0x1157030, device=device@entry=0x115a180)
    at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-device-manager.c:212
        subsystem = <optimized out>
        renderer_type = PLY_RENDERER_TYPE_FRAME_BUFFER
        for_local_console = 64
        device_path = 0x115d570 "/dev/fb0"
        terminal = 0x0
        __func__ = "create_seat_for_udev_device"
#5  0x00007f32215e07a4 in create_seats_for_subsystem (manager=manager@entry=0x1157030, subsystem=subsystem@entry=0x7f32215e8aa3 "graphics")
    at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-device-manager.c:324
        node = 0x115d570 "/dev/fb0"
        device = 0x115a180
        path = <optimized out>
        matches = 0x115cc30
        entry = 0x115a100
        found_device = true
        __func__ = "create_seats_for_subsystem"
#6  0x00007f32215e090f in create_seats_from_udev (manager=0x1157030) at /usr/src/debug/plymouth-0.8.9/src/libply-splash-core/ply-device-manager.c:708
        found_drm_device = true
        found_fb_device = <optimized out>
        __func__ = "create_seats_from_udev"
#7  0x00007f32215f7b34 in ply_event_loop_handle_met_status_for_source (status=PLY_EVENT_LOOP_FD_STATUS_HAS_DATA, source=0x1158b90, loop=0x1151150)
    at /usr/src/debug/plymouth-0.8.9/src/libply/ply-event-loop.c:1060
        next_node = 0x0
        destination = 0x1158ab0
        node = <optimized out>
#8  ply_event_loop_process_pending_events (loop=loop@entry=0x1151150) at /usr/src/debug/plymouth-0.8.9/src/libply/ply-event-loop.c:1336
        source = 0x1158b90
        status = PLY_EVENT_LOOP_FD_STATUS_HAS_DATA
        is_disconnected = <optimized out>
        number_of_received_events = 1
        i = 0
        events = {{events = 1, data = {ptr = 0x1158b90, fd = 18189200, u32 = 18189200, u64 = 18189200}}, {events = 4294967295, data = {ptr = 0xffffffffffffffff, fd = -1, 
              u32 = 4294967295, u64 = 18446744073709551615}} <repeats 63 times>}
        __PRETTY_FUNCTION__ = "ply_event_loop_process_pending_events"
#9  0x00007f32215f7b8f in ply_event_loop_run (loop=0x1151150) at /usr/src/debug/plymouth-0.8.9/src/libply/ply-event-loop.c:1368
No locals.
#10 0x0000000000405e37 in main (argc=<optimized out>, argv=0x7fff37371138) at /usr/src/debug/plymouth-0.8.9/src/main.c:2343
        state = {loop = 0x1151150, boot_server = 0x1153ae0, boot_splash = 0x0, session = 0x1153c00, boot_buffer = 0x1153de0, progress = 0x1156e40, 
          keystroke_triggers = 0x1152a50, entry_triggers = 0x1152a70, entry_buffer = 0x1152a90, messages = 0x1153ac0, command_parser = 0x1151010, mode = PLY_MODE_BOOT, 
          local_console_terminal = 0x11584d0, device_manager = 0x1157030, deactivate_trigger = 0x0, quit_trigger = 0x0, start_time = 38.973372886999996, splash_delay = 5, 
          kernel_command_line = "BOOT_IMAGE=/boot/vmlinuz-3.13.6-nrjQL-desktop-1omv root=UUID=47025b0d-ae8a-47ca-a79d-802aecb6f246 ro quiet splash=silent logo.nologo acpi_osi=Linux acpi_backlight=vendor resume=/dev/sda5 init=/lib/sys"..., kernel_command_line_is_set = 1, no_boot_log = 0, showing_details = 0, system_initialized = 1, 
          is_redirected = 1, is_attached = 1, should_be_attached = 1, should_retain_splash = 0, is_inactive = 0, is_shown = 1, should_force_details = 0, 
          override_splash_path = 0x0, system_default_splash_path = 0x1158920 "/usr/share/plymouth/themes/OpenMandriva/OpenMandriva.plymouth", 
          distribution_default_splash_path = 0x1158be0 "/usr/share/plymouth/themes/spinner/spinner.plymouth", default_tty = 0x40dd3a "/dev/tty1", number_of_errors = 0}
        exit_code = <optimized out>
        should_help = false
        no_daemon = false
        debug = false
        attach_to_session = true
        daemon_handle = <optimized out>
        mode_string = 0x1151960 "\200"
        kernel_command_line = 0x0
        tty = 0x0
        device_manager_flags = PLY_DEVICE_MANAGER_FLAGS_NONE
        __func__ = "main"

## Comment 4138
Date: 2014-03-20 08:17:26 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

uname -a
Linux localhost.localdomain 3.13.6-nrjQL-desktop-1omv #1 SMP PREEMPT Thu Mar 13 10:55:37 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

alpha 2 after all update and  dracut -f

# rpm -qa --last | grep plymouth
plymouth-system-theme-0.8.9-0.20140316.4-omv2014.0.x86_64 jeu. 20 mars 2014 08:03:50 CET
plymouth-plugin-script-0.8.9-0.20140316.4-omv2014.0.x86_64 jeu. 20 mars 2014 08:03:50 CET
plymouth-utils-0.8.9-0.20140316.4-omv2014.0.x86_64 jeu. 20 mars 2014 08:03:49 CET
plymouth-plugin-label-0.8.9-0.20140316.4-omv2014.0.x86_64 jeu. 20 mars 2014 08:03:48 CET
plymouth-scripts-0.8.9-0.20140316.4-omv2014.0.x86_64 jeu. 20 mars 2014 08:03:47 CET
plymouth-0.8.9-0.20140316.4-omv2014.0.x86_64  jeu. 20 mars 2014 08:03:46 CET

boot:

first: systemd-fsck (at each boot !) on all disks

after : plymouth seems to  work but there is not a big circle as  before, simply 3  small white squares on the middle of the screen during 3,4 seconds

after:little circle on the middle of the screen

then log-screen

## Comment 4139
Date: 2014-03-20 08:19:28 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 286
boot with plymouth 0.8.9-0.20140316.4-omv2014.0

journalctl -b

> Attached file: boot_plymouth-0.8.9-0.20140316.4-omv2014.0.x86_64.txt (text/plain, 119276 bytes)
> Description:   boot with plymouth 0.8.9-0.20140316.4-omv2014.0

## Comment 4140
Date: 2014-03-20 08:21:03 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 287
dracut with plymouth-0.8.9-0.20140316.4

dracut -f

> Attached file: dracut_plymouth-0.8.9-0.20140316.4.txt (text/plain, 2271 bytes)
> Description:   dracut with plymouth-0.8.9-0.20140316.4

## Comment 4142
Date: 2014-03-20 08:51:12 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

in addition
# systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: inactive (dead) since jeu. 2014-03-20 08:21:16 CET; 1h 28min ago
  Process: 429 ExecStartPost=/bin/plymouth show-splash (code=exited, status=0/SUCCESS)
  Process: 420 ExecStart=/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session (code=exited, status=0/SUCCESS)
 Main PID: 424 (code=exited, status=0/SUCCESS)

mars 20 08:21:06 localhost.localdomain systemd[1]: Started Show Plymouth Boot Screen.

## Comment 4143
Date: 2014-03-20 09:38:24 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 288
info and core file

in my vbox still core dumped

> Attached file: coredumped.txt (text/plain, 6575 bytes)
> Description:   info and core file

## Comment 4152
Date: 2014-03-20 15:08:09 +0000
From: @itchka

The solution to this may be embarrassingly simple. Try removing init=/usr/lib/systemd from your grub command line. Works for me an eeepc.

## Comment 4154
Date: 2014-03-20 15:31:00 +0000
From: @itchka

correction init=/lib/systend/systemd

## Comment 4155
Date: 2014-03-20 15:40:26 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

alpha 2 

]# cat /proc/cmdline
BOOT_IMAGE=/boot/vmlinuz-3.13.6-nrjQL-desktop-1omv root=UUID=5b3a8a86-a90a-4fad-8bd9-48b35b6cd07f ro quiet splash=silent logo.nologo acpi_osi=Linux acpi_backlight=vendor resume=/dev/sdb5

I have no more systemd-fsck aon the screen before boot

I have big circle plymouth but a short time

systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: inactive (dead) since jeu. 2014-03-20 16:26:08 CET; 6min ago
  Process: 563 ExecStartPost=/bin/plymouth show-splash (code=exited, status=0/SUCCESS)
  Process: 542 ExecStart=/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session (code=exited, status=0/SUCCESS)
 Main PID: 562 (code=exited, status=0/SUCCESS)

## Comment 4156
Date: 2014-03-20 16:01:50 +0000
From: @itchka

Joel,
Check for the following in your /etc/plymouth/plymouthd.conf
also in /usr/share/plymouthd.defaults. I have both files the same.

[Daemon]
Theme=OpenMandriva
ShowDelay=0

ShowDelay is a new parameter which delays the splash the default is 5 seconds if your machine is fast you may only get a short splash.

You should rebuild your initramfs to get the benefit.

So as root  dracut -f

and then reboot.

## Comment 4157
Date: 2014-03-20 17:46:40 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #41)
> Joel,
> Check for the following in your /etc/plymouth/plymouthd.conf
> also in /usr/share/plymouthd.defaults. I have both files the same.
> 
> [Daemon]
> Theme=OpenMandriva
> ShowDelay=0
> 
> ShowDelay is a new parameter which delays the splash the default is 5
> seconds if your machine is fast you may only get a short splash.
> 
> You should rebuild your initramfs to get the benefit.
> 
> So as root  dracut -f
> 
> and then reboot.

Colin,
I did a lot of tries ,every time whith dracut -f and boot line without "init=/lib/systemd/systemd"
just changed /etc/plymouth/plymouthd.conf with ShowDelay=10,0,5,7
i have always  systemd-fsck at the beginning of the boot  except with delay=0
with 10 no big circle
with 5 or 7 big circle but not for long

this does not solve the problem, sorry 

(# cat /usr/share/plymouth/plymouthd.defaults
# Distribution defaults. Changes to this file will get overwritten during
# upgrades.
[Daemon]
Theme=spinner
)

## Comment 4158
Date: 2014-03-20 18:00:37 +0000
From: @tpgxyz

First make sure your are free of "init=/lib/systemd/systemd" declarations in /etc/default/grub
/etc/grub2.cfg
/etc/grub.d/*

If you remove these declaration then start dracut -f

This solved for me issue with plymouth blocking boot on three machines
1. AMD 4xPhenom
2. Tosiba Satllite
3. Dell e6400

New fixed grub is being bublishing with removed declarations of init.

## Comment 4180
Date: 2014-03-20 22:59:33 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

build 2991 
no plymouth, no gear still coredump in vbox

# systemctl --failed
UNIT                   LOAD   ACTIVE SUB    DESCRIPTION
plymouth-start.service loaded failed failed Show Plymouth Boot Screen

LOAD   = Reflects whether the unit definition was properly loaded.
ACTIVE = The high-level unit activation state, i.e. generalization of SUB.
SUB    = The low-level unit activation state, values depend on unit type.

1 loaded units listed. Pass --all to see loaded but inactive units, too.
To show all installed unit files use 'systemctl list-unit-files'.

# systemctl status plymouth-start.service
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: failed (Result: core-dump) since gio 2014-03-20 23:49:39 CET; 2min 35s ago
 Main PID: 446 (code=dumped, signal=SEGV)

mar 20 23:49:37 localhost.localdomain systemd[1]: Started Show Plymouth Boot ...
mar 20 23:49:39 localhost.localdomain systemd-coredump[1176]: Process 446 (pl...
mar 20 23:49:39 localhost.localdomain systemd[1]: plymouth-start.service: mai...
mar 20 23:49:39 localhost.localdomain systemd[1]: Unit plymouth-start.service...
Hint: Some lines were ellipsized, use -l to show in full.

## Comment 4184
Date: 2014-03-20 23:48:43 +0000
From: @christann404

removing the init=/lib/systemd/systemd worked for me, but even after the last update to plymouth and grub, I still had to remove it by hand.

## Comment 4190
Date: 2014-03-21 05:11:23 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to comment #43)

> 
> New fixed grub is being bublishing with removed declarations of init.

after your updates, update-grub2 and dracut -f

alpha 2 real installation

# cat /proc/cmdline
BOOT_IMAGE=/boot/vmlinuz-3.13.6-nrjQL-desktop-1omv root=UUID=5b3a8a86-a90a-4fad-8bd9-48b35b6cd07f ro quiet splash=silent logo.nologo acpi_osi=Linux acpi_backlight=vendor resume=/dev/sdb5

# cat /etc/plymouth/plymouthd.conf
# Administrator customizations go in this file
#[Daemon]
#Theme=fade-in
ShowDelay=0

# systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: inactive (dead) since ven. 2014-03-21 05:46:34 CET; 7min ago
  Process: 408 ExecStartPost=/bin/plymouth show-splash (code=exited, status=0/SUCCESS)
  Process: 387 ExecStart=/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session (code=exited, status=0/SUCCESS)
 Main PID: 388 (code=exited, status=0/SUCCESS)

mars 21 05:46:23 localhost.localdomain systemd[1]: Started Show Plymouth Boot Screen.
# systemctl --failed
0 loaded units listed.

but 

- always systemd-fsck on the screen (very irritating) :=)
- no big circle

## Comment 4251
Date: 2014-03-22 10:31:05 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

alpha 2 real installation

rpm -qa --last | grep dracut
dracut-036-4-omv2014.0.x86_64                 sam. 22 mars 2014 11:02:33 CET- 

always systemd-fsck on the screen 
- no big circle
but I am logging and the system is working

## Comment 4252
Date: 2014-03-22 10:33:19 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 304
omv-bug-report with new dracut

my system (omv-bug-report) with dracut-036-4-omv2014.0.x86_64

> Attached file: report_with_new_dracut.xz (application/x-xz, 59564 bytes)
> Description:   omv-bug-report with new dracut

## Comment 4253
Date: 2014-03-22 10:57:15 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

in vbox 
build 2991

i can boot and login

but i have no gear and plymouthd coredump

# systemd-coredumpctl 
TIME                                         PID   UID   GID SIG EXE
              gio 2014-03-20 23:49:39 CET    446     0     0  11 /sbin/plymouthd
              ven 2014-03-21 00:19:35 CET   8758     0     0  11 /sbin/plymouthd
              ven 2014-03-21 00:19:59 CET    432     0     0  11 /sbin/plymouthd
              ven 2014-03-21 06:47:10 CET   4441     0     0  11 /sbin/plymouthd
              ven 2014-03-21 07:20:18 CET   5331     0     0  11 /sbin/plymouthd
              ven 2014-03-21 09:21:27 CET    436     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:19:02 CET  16311     0     0  11 /sbin/plymouthd
              ven 2014-03-21 10:54:28 CET    436     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:31:19 CET  21339     0     0  11 /sbin/plymouthd
              ven 2014-03-21 11:06:48 CET    438     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:38:17 CET   5649     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:40:03 CET   4699     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:42:05 CET    408     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:47:50 CET  16468     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:48:09 CET    441     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:52:13 CET  16212     0     0  11 /sbin/plymouthd
              ven 2014-03-21 14:54:02 CET    430     0     0  11 /sbin/plymouthd
              sab 2014-03-22 08:49:23 CET  31285     0     0  11 /sbin/plymouthd
              ven 2014-03-21 18:23:55 CET    435     0     0  11 /sbin/plymouthd

## Comment 4254
Date: 2014-03-22 10:58:04 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 305
omv-bug-report

> Attached file: omv-bug-report.log.xz (application/octet-stream, 53852 bytes)
> Description:   omv-bug-report

## Comment 4257
Date: 2014-03-22 12:04:34 +0000
From: @rugyada

pre-Beta build 2989
VirtualBox virtual machine 64bit

# systemd-coredumpctl
TIME                                         PID   UID   GID SIG EXE
              Fri 2014-03-21 01:30:34 CET    455     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 03:04:46 CET  16150     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 03:05:00 CET    436     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 03:08:00 CET   4013     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 10:03:34 CET    468     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 10:08:09 CET   4660     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 10:10:51 CET    441     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 10:34:01 CET   7691     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 10:35:43 CET    445     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 11:27:28 CET  24953     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 18:55:07 CET    453     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 19:01:09 CET   6942     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 19:03:44 CET    420     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 19:15:02 CET   8698     0     0  11 /sbin/plymouthd
              Fri 2014-03-21 19:18:22 CET    466     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:06:08 CET  17950     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:07:49 CET    435     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:09:40 CET   6040     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:13:57 CET    424     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:14:45 CET   4884     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:16:28 CET    455     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:17:41 CET   5739     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:17:56 CET    435     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:18:25 CET   4499     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:18:40 CET    431     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:19:56 CET   4687     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 01:24:35 CET   5489     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 09:33:36 CET    476     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 10:03:26 CET  29413     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 10:05:17 CET    458     0     0  11 /sbin/plymouthd
              Sat 2014-03-22 10:13:32 CET  15741     0     0  11 /sbin/plymouthd

## Comment 4258
Date: 2014-03-22 12:19:33 +0000
From: @itchka

Hi luca,
Can you try removing any references to resume= from your kernel commandline.
Thanks,
Colin

Colin Close
QA Team

## Comment 4259
Date: 2014-03-22 13:02:12 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Hi Colin,
tried but with no luck:(
No gear, only 3 white dots for a second on black screen and then start
textual boot logs

plymouth-start.service coredump

system boots and i can login

=================================================================
$ more /proc/cmdline 
BOOT_IMAGE=/boot/vmlinuz-3.13.6-nrjQL-desktop-1omv root=UUID=a14d3a82-3fe8-4bf3-bb10-2f5908971b00 ro quiet splash=silent logo.nologo acpi_osi=Linux
 acpi_backlight=vendor

=================================================================
# systemctl status plymouth-start.service 
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: failed (Result: core-dump) since sab 2014-03-22 13:54:08 CET; 2min 41s ago
  Process: 472 ExecStartPost=/bin/plymouth show-splash (code=exited, status=0/SUCCESS)
  Process: 421 ExecStart=/sbin/plymouthd --mode=boot --pid-file=/run/plymouth/pid --attach-to-session (code=exited, status=0/SUCCESS)
 Main PID: 454 (code=dumped, signal=SEGV)

mar 22 13:54:06 localhost.localdomain systemd[1]: Started Show Plymouth Boot Screen.
mar 22 13:54:08 localhost.localdomain systemd-coredump[1258]: Process 454 (plymouthd) dumped core.
mar 22 13:54:08 localhost.localdomain systemd[1]: plymouth-start.service: main process exited, code=dumped, status=11/SEGV
mar 22 13:54:08 localhost.localdomain systemd[1]: Unit plymouth-start.service entered failed state.

## Comment 4260
Date: 2014-03-22 16:05:04 +0000
From: @tpgxyz

Can you please add here attachement with  output form systemd-coredumpctl PID
where PID is a last pid of plymouth for systemd-coredumpctl list.
Please install also all needed debuginfo packages. Without detailed information is hard to get any clue what can be wrong in your case. Maybe it is somehow related to you specific hardware.

Have you ever tried to boot from CD phisically on your machine, without virtualbox, as well know is that virtualbox can produce varius issues which does not exists on phisicall hardware.

Please attach proper output from omv-buf-report.sh, because above files are empty.

## Comment 4262
Date: 2014-03-22 16:21:49 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to comment #54)
> Can you please add here attachement with  output form systemd-coredumpctl PID
> where PID is a last pid of plymouth for systemd-coredumpctl list.
> Please install also all needed debuginfo packages. Without detailed
> information is hard to get any clue what can be wrong in your case. Maybe it
> is somehow related to you specific hardware.

would be the third time I send the stack trace of the core dump

> 
> Have you ever tried to boot from CD phisically on your machine, without
> virtualbox, as well know is that virtualbox can produce varius issues which
> does not exists on phisicall hardware.

i have no phisicall hardware available
Have you ever tried to install a vbox and try an installation of oma?
you need half an hour

> 
> Please attach proper output from omv-buf-report.sh, because above files are
> empty.

strange. 
check your tool.
have you used unxz?

i have downloaded the attachment and it is not empty.

## Comment 4263
Date: 2014-03-22 16:23:22 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Created attachment 306
omv.bug.report.log

> Attached file: omv-bug-report.log.txt (text/plain, 263648 bytes)
> Description:   omv.bug.report.log

## Comment 4264
Date: 2014-03-22 16:31:58 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to comment #54)
comment 32 for an output with debug info installed

> Maybe it  is somehow related to you specific hardware.

but it's a virtual machine. 
what does my hardware

rugyada has same plymouth coredump

## Comment 4267
Date: 2014-03-22 17:42:50 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

real installation beta 64 2989

direct installation

after installation and first reboot passwd root, user etc

reboot:first black screen and 3 squares in the middle of screen during 5-6 s

no plymouth screen

little circle 3-4 s and log screen

all is ok I have omv screen 

quick shutdown for the moment

[root@localhost moi]# systemd-analyze critical-chain
The time after the unit is active or started is printed after the "@" character.
The time the unit takes to start is printed after the "+" character.

graphical.target @25.055s
└─multi-user.target @25.054s
  └─smb.service @24.563s +490ms
    └─winbind.service @23.592s +959ms
      └─nmb.service @23.371s +217ms
        └─network.target @23.094s
          └─network.service @19.830s +3.263s
            └─NetworkManager.service @18.897s +927ms
              └─firewalld.service @10.669s +8.223s
                └─basic.target @10.615s
                  └─mandriva-everytime.service @9.802s +794ms
                    └─local-fs.target @9.800s
                      └─rosa.mount @9.590s +162ms
                        └─systemd-fsck@dev-disk-by\x2duuid-afbb2845\x2d6b04\x2d4564\x2db638\x2d70f073f65d34.service @8.376s +1.079s
                          └─dev-disk-by\x2duuid-afbb2845\x2d6b04\x2d4564\x2db638\x2d70f073f65d34.device @8.350s

# systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: inactive (dead) since sam. 2014-03-22 18:01:41 CET; 38min ago
 Main PID: 281 (code=exited, status=0/SUCCESS)

mars 22 19:01:26 localhost systemd[1]: Starting Show Plymouth Boot Screen...
mars 22 19:01:26 localhost systemd[1]: Started Show Plymouth Boot Screen.

## Comment 4269
Date: 2014-03-22 17:44:18 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 307
install beta 64 2989

after install beta 64 2989

> Attached file: report_install_beta.xz (application/x-xz, 59168 bytes)
> Description:   install beta 64 2989

## Comment 4276
Date: 2014-03-23 00:50:13 +0000
From: @robxu9

Created attachment 308
omv-bug-report.sh bug report log

I'm still core-dumping myself, even with uvesafb gone and my /proc/cmdline clean of any init=/lib/systemd/systemd references.

This is on a real machine, Dell Inspiron 14z, as well as KVM.

> Attached file: omv-bug-report.log.xz (application/x-xz, 51952 bytes)
> Description:   omv-bug-report.sh bug report log

## Comment 4277
Date: 2014-03-23 00:54:19 +0000
From: @robxu9

Resulting coredump from me: https://dl.dropboxusercontent.com/u/3635097/_sbin_plymouthd_coredump

## Comment 4278
Date: 2014-03-23 01:42:45 +0000
From: @robxu9

(In reply to comment #60)
> Created attachment 308 [details]
> omv-bug-report.sh bug report log
> 
> I'm still core-dumping myself, even with uvesafb gone and my /proc/cmdline
> clean of any init=/lib/systemd/systemd references.
> 
> This is on a real machine, Dell Inspiron 14z, as well as KVM.

Actually, disregard this. itchka and I managed to investigate and we came to the conclusion that it's just my computer &/or VM acting up.

## Comment 4288
Date: 2014-03-23 19:34:55 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

new beta  x64
real installation
 
I have plymouth on screen
seems normal
system works

#  systemd-analyze time
Startup finished in 3.385s (kernel) + 4.044s (initrd) + 19.149s (userspace) = 26.579s

# systemctl status plymouth-start
plymouth-start.service - Show Plymouth Boot Screen
   Loaded: loaded (/lib/systemd/system/plymouth-start.service; disabled)
   Active: inactive (dead) since dim. 2014-03-23 20:16:45 CET; 12min ago
 Main PID: 264 (code=exited, status=0/SUCCESS)

mars 23 21:16:28 localhost systemd[1]: Started Show Plymouth Boot Screen.

CORRECTION :just now  in journalctl a lot of debug info:

mars 23 19:50:10 localhost.localdomain 40grub2[7279]: debug: parsing: else
mars 23 19:50:10 localhost.localdomain 40grub2[7280]: debug: parsing: search --no-floppy --fs-uuid --set=root afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:10 localhost.localdomain 40grub2[7281]: debug: parsing: fi
mars 23 19:50:10 localhost.localdomain 40grub2[7282]: debug: parsing: linux /boot/vmlinuz-3.10.19-nrj-desktop-2rosa root=UUID=afbb2845-6b04-4564-b638-70f073f65d34 ro splash=silent logo.nologo nomodeset resume= init=/lib/systemd/systemd
mars 23 19:50:10 localhost.localdomain 40grub2[7286]: debug: parsing: initrd /boot/initrd-3.10.19-nrj-desktop-2rosa.img
mars 23 19:50:11 localhost.localdomain 40grub2[7290]: debug: parsing: }
mars 23 19:50:11 localhost.localdomain 40grub2[7291]: debug: parsing: menuentry 'ROSA Desktop Fresh R2, avec Linux 3.10.19-desktop-2rosa (sur /dev/sda2)' --class rosadesktop.fresh --class gnu-linux --class gnu --class os $menuentry_id_option 'osprober-gnulinux-/boot/vmlinuz-3.10.19-desktop-2rosa--afbb2845-6b04-4564-b638-70f073f65d34' {
mars 23 19:50:11 localhost.localdomain 40grub2[7302]: debug: parsing: insmod part_msdos
mars 23 19:50:11 localhost.localdomain 40grub2[7303]: debug: parsing: insmod ext2
mars 23 19:50:11 localhost.localdomain 40grub2[7304]: debug: parsing: set root='hd0,msdos2'
mars 23 19:50:11 localhost.localdomain 40grub2[7305]: debug: parsing: if [ x$feature_platform_search_hint = xy ]; then
mars 23 19:50:11 localhost.localdomain 40grub2[7306]: debug: parsing: search --no-floppy --fs-uuid --set=root --hint-bios=hd0,msdos2 --hint-efi=hd0,msdos2 --hint-baremetal=ahci0,msdos2  afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7307]: debug: parsing: else
mars 23 19:50:11 localhost.localdomain 40grub2[7308]: debug: parsing: search --no-floppy --fs-uuid --set=root afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7309]: debug: parsing: fi
mars 23 19:50:11 localhost.localdomain 40grub2[7310]: debug: parsing: linux /boot/vmlinuz-3.10.19-desktop-2rosa root=UUID=afbb2845-6b04-4564-b638-70f073f65d34 ro splash=silent logo.nologo nomodeset resume= init=/lib/systemd/systemd
mars 23 19:50:11 localhost.localdomain 40grub2[7314]: debug: parsing: initrd /boot/initrd-3.10.19-desktop-2rosa.img
mars 23 19:50:11 localhost.localdomain 40grub2[7318]: debug: parsing: }
mars 23 19:50:11 localhost.localdomain 40grub2[7319]: debug: parsing: menuentry 'ROSA Desktop Fresh R2, avec Linux 3.10.9-nrj-desktop-1rosa (sur /dev/sda2)' --class rosadesktop.fresh --class gnu-linux --class gnu --class os $menuentry_id_option 'osprober-gnulinux-/boot/vmlinuz-3.10.9-nrj-desktop-1rosa--afbb2845-6b04-4564-b638-70f073f65d34' {
mars 23 19:50:11 localhost.localdomain 40grub2[7330]: debug: parsing: insmod part_msdos
mars 23 19:50:11 localhost.localdomain 40grub2[7331]: debug: parsing: insmod ext2
mars 23 19:50:11 localhost.localdomain 40grub2[7332]: debug: parsing: set root='hd0,msdos2'
mars 23 19:50:11 localhost.localdomain 40grub2[7333]: debug: parsing: if [ x$feature_platform_search_hint = xy ]; then
mars 23 19:50:11 localhost.localdomain 40grub2[7334]: debug: parsing: search --no-floppy --fs-uuid --set=root --hint-bios=hd0,msdos2 --hint-efi=hd0,msdos2 --hint-baremetal=ahci0,msdos2  afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7335]: debug: parsing: else
mars 23 19:50:11 localhost.localdomain 40grub2[7336]: debug: parsing: search --no-floppy --fs-uuid --set=root afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7337]: debug: parsing: fi
mars 23 19:50:11 localhost.localdomain 40grub2[7338]: debug: parsing: linux /boot/vmlinuz-3.10.9-nrj-desktop-1rosa root=UUID=afbb2845-6b04-4564-b638-70f073f65d34 ro splash=silent logo.nologo nomodeset resume= init=/lib/systemd/systemd
mars 23 19:50:11 localhost.localdomain 40grub2[7342]: debug: parsing: initrd /boot/initrd-3.10.9-nrj-desktop-1rosa.img
mars 23 19:50:11 localhost.localdomain 40grub2[7346]: debug: parsing: }
mars 23 19:50:11 localhost.localdomain 40grub2[7347]: debug: parsing: menuentry 'ROSA Desktop Fresh R2, avec Linux 3.6.10-nrj-desktop-1rosa (sur /dev/sda2)' --class rosadesktop.fresh --class gnu-linux --class gnu --class os $menuentry_id_option 'osprober-gnulinux-/boot/vmlinuz-3.6.10-nrj-desktop-1rosa--afbb2845-6b04-4564-b638-70f073f65d34' {
mars 23 19:50:11 localhost.localdomain 40grub2[7358]: debug: parsing: insmod part_msdos
mars 23 19:50:11 localhost.localdomain 40grub2[7359]: debug: parsing: insmod ext2
mars 23 19:50:11 localhost.localdomain 40grub2[7360]: debug: parsing: set root='hd0,msdos2'
mars 23 19:50:11 localhost.localdomain 40grub2[7361]: debug: parsing: if [ x$feature_platform_search_hint = xy ]; then
mars 23 19:50:11 localhost.localdomain 40grub2[7362]: debug: parsing: search --no-floppy --fs-uuid --set=root --hint-bios=hd0,msdos2 --hint-efi=hd0,msdos2 --hint-baremetal=ahci0,msdos2  afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7363]: debug: parsing: else
mars 23 19:50:11 localhost.localdomain 40grub2[7364]: debug: parsing: search --no-floppy --fs-uuid --set=root afbb2845-6b04-4564-b638-70f073f65d34
mars 23 19:50:11 localhost.localdomain 40grub2[7365]: debug: parsing: fi
mars 23 19:50:11 localhost.localdomain 40grub2[7366]: debug: parsing: linux /boot/vmlinuz-3.6.10-nrj-desktop-1rosa root=UUID=afbb2845-6b04-4564-b638-70f073f65d34 ro splash=silent logo.nologo nomodeset resume= init=/lib/systemd/systemd
mars 23 19:50:11 localhost.localdomain 40grub2[7370]: debug: parsing: initrd /boot/initrd-3.6.10-nrj-desktop-1rosa.img
mars 23 19:50:11 localhost.localdomain 40grub2[7374]: debug: parsing: }
mars 23 19:50:11 localhost.localdomain 40grub2[7375]: debug: parsing: }
mars 23 19:50:11 localhost.localdomain 40grub2[7376]: debug: parsing:
mars 23 19:50:11 localhost.localdomain 40grub2[7377]: debug: parsing: ### END /etc/grub.d/30_os-prober ###
mars 23 19:50:11 localhost.localdomain 40grub2[7378]: debug: parsing:
mars 23 19:50:11 localhost.localdomain 40grub2[7379]: debug: parsing: ### BEGIN /etc/grub.d/40_custom ###
mars 23 19:50:11 localhost.localdomain 40grub2[7380]: debug: parsing: # This file provides an easy way to add custom menu entries.  Simply type the
mars 23 19:50:11 localhost.localdomain 40grub2[7381]: debug: parsing: # menu entries you want to add after this comment.  Be careful not to change
mars 23 19:50:11 localhost.localdomain 40grub2[7382]: debug: parsing: # the 'exec tail' line above.
mars 23 19:50:11 localhost.localdomain 40grub2[7383]: debug: parsing: ### END /etc/grub.d/40_custom ###
mars 23 19:50:11 localhost.localdomain 40grub2[7384]: debug: parsing:
mars 23 19:50:11 localhost.localdomain 40grub2[7385]: debug: parsing: ### BEGIN /etc/grub.d/41_custom ###
mars 23 19:50:11 localhost.localdomain 40grub2[7386]: debug: parsing: if [ -f  ${config_directory}/custom.cfg ]; then
mars 23 19:50:11 localhost.localdomain 40grub2[7387]: debug: parsing: source ${config_directory}/custom.cfg
mars 23 19:50:11 localhost.localdomain 40grub2[7388]: debug: parsing: elif [ -z "${config_directory}" -a -f  $prefix/custom.cfg ]; then
mars 23 19:50:11 localhost.localdomain 40grub2[7389]: debug: parsing: source $prefix/custom.cfg;
mars 23 19:50:11 localhost.localdomain 40grub2[7390]: debug: parsing: fi
mars 23 19:50:11 localhost.localdomain 40grub2[7391]: debug: parsing: ### END /etc/grub.d/41_custom ###
mars 23 19:50:11 localhost.localdomain 40grub2[7392]: debug: parsing:
mars 23 19:50:11 localhost.localdomain 40grub2[7393]: debug: parsing: ### BEGIN /etc/grub.d/90_persistent ###
mars 23 19:50:11 localhost.localdomain 40grub2[7394]: debug: parsing: ### END /etc/grub.d/90_persistent ###
mars 23 19:50:11 localhost.localdomain linux-boot-prober[7395]: debug: /usr/libexec/linux-boot-probes/mounted/40grub2 succeeded
mars 23 19:50:11 localhost.localdomain draklive-install[3682]: launched command: umount /mnt/install/proc; umount /mnt/install/sys
mars 23 19:50:12 localhost.localdomain udisks-daemon[686]: **** /proc/self/mountinfo changed
mars 23 19:50:13 localhost.localdomain draklive-install[3682]: launched command: sync
-- Reboot --

## Comment 4289
Date: 2014-03-23 19:44:31 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

Created attachment 310
journactl --boot=-3

journal with debug info beta x84 3015

> Attached file: journalctl_beta3015.txt (text/plain, 313330 bytes)
> Description:   journactl --boot=-3

## Comment 4293
Date: 2014-03-24 05:03:06 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

beta 3015 x64 real installation 

I redid a new installation, even messages terminated with ;

mars 24 05:14:29 localhost.localdomain 40grub2[7404]: debug: parsing: ### END /etc/grub.d/90_persistent ###
mars 24 05:14:29 localhost.localdomain linux-boot-prober[7405]: debug: /usr/libexec/linux-boot-probes/mounted/40grub2 succeeded

but I boot normally 
plymouth ok

# systemd-analyze time
Startup finished in 3.386s (kernel) + 3.941s (initrd) + 20.801s (userspace) = 28.129s

## Comment 4423
Date: 2014-03-25 12:39:11 +0000
From: @itchka

Reduced the importance of this bug as the major issue of not booting has been resolved. A short period of black screen or the non-display of the bootsplash is considered acceptable at this time. The problem has being investigated and the cause is largely due to the new kernels very fast boot times. Upstream are investigating a solution.

## Comment 5325
Date: 2014-04-22 15:27:46 +0000
From: @itchka

See previous entry
